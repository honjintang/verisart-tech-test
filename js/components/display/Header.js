import React from 'react'
import VerisartLogo from '../../../assets/images/verisartLogo.svg'

const Header = () => (
    <div className="head">
        <div className="gdprBanner">
            We use cookies to improve your user experience on our site. By using Verisart you agree to our 
            <a href="/privacy"> cookies and privacy policy</a>. Got it.
        </div>
        <div className="nav">
            <div className="logoWrapper">
                <a href="/"><img className="logo" src={VerisartLogo}></img></a>
            </div>
            <nav className='navBar'>
                    <div className="navOption"><a href="/news">news</a></div>
                    <div className="navOption"><a href="/partners">partners</a></div>
                    <div className="navOption"><a href="/join">join</a></div>
                    <div className="navOption"><a href="/login">log in</a></div>
            </nav>
        </div>
    </div>
)

export default Header