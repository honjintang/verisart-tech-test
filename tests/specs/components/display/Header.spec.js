import React from 'react'
import { shallow } from 'enzyme'
import Header from '~/components/display/Header'

describe('components/display/Header', () => {
    it('matches snapshot', () => {
        const component = shallow(<Header />)
        expect(component).toMatchSnapshot()
    })
})