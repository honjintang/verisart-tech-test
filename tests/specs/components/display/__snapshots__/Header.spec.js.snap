// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/display/Header matches snapshot 1`] = `
<div
  className="head"
>
  <div
    className="gdprBanner"
  >
    We use cookies to improve your user experience on our site. By using Verisart you agree to our
    <a
      href="/privacy"
    >
       cookies and privacy policy
    </a>
    . Got it.
  </div>
  <div
    className="nav"
  >
    <div
      className="logoWrapper"
    >
      <a
        href="/"
      >
        <img
          className="logo"
          src="test-file-stub"
        />
      </a>
    </div>
    <nav
      className="navBar"
    >
      <div
        className="navOption"
      >
        <a
          href="/news"
        >
          news
        </a>
      </div>
      <div
        className="navOption"
      >
        <a
          href="/partners"
        >
          partners
        </a>
      </div>
      <div
        className="navOption"
      >
        <a
          href="/join"
        >
          join
        </a>
      </div>
      <div
        className="navOption"
      >
        <a
          href="/login"
        >
          log in
        </a>
      </div>
    </nav>
  </div>
</div>
`;
